UPDATING.md

```diff
@@ -24,6 +24,34 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Change of method signature in `GCPTransferServiceHook`
+
+The signature of the `create_transfer_job` method in `GCPTransferServiceHook`
+class has changed. The change does not change the behavior of the method.
+Old signature:
+```python
+def create_transfer_job(self, description, schedule, transfer_spec, project_id=None):
+```
+New signature:
+```python
+def create_transfer_job(self, body):
+```
+
+It is necessary to rewrite calls to method. The new call looks like this:
+```python
+body = {
+'status': 'ENABLED',
+  'projectId': project_id,
+  'description': description,
+  'transferSpec': transfer_spec,
+  'schedule': schedule,
+}
+gct_hook.create_transfer_job(body)
+```
+The change results from the unification of all hooks and getting to
```

Comment:
```suggestion
The change results from the unification of all hooks and adjust to
```
Ktoś z angielskim bardziej zaprzyjażniony powinien zdecydować, jak to lepiej brzmi.

Reply:
adjust brzmi lepiej :)